flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/DataIterator.java

```diff
@@ -41,16 +42,49 @@
   private final FileScanTaskReader<T> fileScanTaskReader;
 
   private final InputFilesDecryptor inputFilesDecryptor;
-  private Iterator<FileScanTask> tasks;
+  private final CombinedScanTask combinedTask;
+
+  private Iterator<FileScanTask> fileTasksIterator;
```

Comment:
Why rename this? Wasn't `tasks` clear? Renaming introduces more changes.

Reply:
because I am adding a `combinedTask` in this PR, I thought it may be clearer to rename `tasks` to `fileTasksIterator`. I can certainly revert it.